utilities/options/options_util_test.cc

```diff
@@ -94,6 +94,45 @@ TEST_F(OptionsUtilTest, SaveAndLoad) {
   }
 }
 
+TEST_F(OptionsUtilTest, SaveAndLoadWithCacheCheck) {
+  // creating db
+  DBOptions db_opt;
+  db_opt.create_if_missing = true;
+  // initialize BlockBasedTableOptions
+  std::shared_ptr<Cache> cache = NewLRUCache(1 * 1024);
+  BlockBasedTableOptions bbt_opts;
+  bbt_opts.block_size = 32 * 1024;
+  // saving cf options
+  std::vector<ColumnFamilyOptions> cf_opts;
+  ColumnFamilyOptions cf_opt_sample1 = ColumnFamilyOptions();
+  cf_opt_sample1.table_factory.reset(NewBlockBasedTableFactory(bbt_opts));
+  cf_opts.push_back(cf_opt_sample1);
+  ColumnFamilyOptions cf_opt_sample2 = ColumnFamilyOptions();
+  cf_opt_sample2.table_factory.reset(NewBlockBasedTableFactory(bbt_opts));
+  cf_opts.push_back(cf_opt_sample2);
```

Comment:
if I'm reading this correctly, `cf_opt_sample1` is for column family `kDefaultColumnFamilyName` and `cf_opt_sample2` is for column family `cf_sample2`? I suggest rename the ColumnFamilyOptions variable to reflect the column family name it corresponds to. Also would you consider varying the table_factory type? Currently both are block based table factory.